### DISCUSSION TITLE
snippets for multiple languages

### DISCUSSION THREAD

**Discussion by jonas-frey** [draivin/hsnips#87] [Q&A] [answered]
Can I create snippets for multiple languages at once? I would like to share snippets between latex and markdown.

**Comment by draivin** [comment, marked as answer]
You can create a file `all.hsnips` with snips that will apply to every language, there is no other way to create shared snippets currently.

**Reply by jonas-frey** [reply]
Can I have several snippet files for the same language? That, together with symbolic links, would also allow to share snippets between multiple languages.
